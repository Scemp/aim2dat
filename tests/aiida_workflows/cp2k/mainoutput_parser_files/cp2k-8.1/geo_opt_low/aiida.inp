!!! Generated by AiiDA !!!
&FORCE_EVAL
   &DFT
      BASIS_SET_FILE_NAME BASIS_MOLOPT
      &MGRID
         CUTOFF 280
         NGRIDS 4
         REL_CUTOFF 30
      &END MGRID
      &POISSON
         PERIODIC none
         PSOLVER MT
      &END POISSON
      POTENTIAL_FILE_NAME GTH_POTENTIALS
      &QS
         EPS_DEFAULT 1e-12
         EXTRAPOLATION_ORDER 3
         WF_INTERPOLATION ps
      &END QS
      &XC
         &XC_FUNCTIONAL PBE
         &END XC_FUNCTIONAL
      &END XC
   &END DFT
   METHOD Quickstep
   &SUBSYS
      &CELL
         A 4.0             0.0             0.0            
         B 0.0             4.0             0.0            
         C 0.0             0.0             4.737166       
         PERIODIC none
      &END CELL
      &KIND O
         BASIS_SET DZVP-MOLOPT-SR-GTH
         POTENTIAL GTH-PBE-q6
      &END KIND
      &KIND H
         BASIS_SET DZVP-MOLOPT-SR-GTH
         POTENTIAL GTH-PBE-q1
      &END KIND
      &TOPOLOGY
         COORD_FILE_FORMAT XYZ
         COORD_FILE_NAME aiida.coords.xyz
      &END TOPOLOGY
   &END SUBSYS
&END FORCE_EVAL
&GLOBAL
   PROJECT aiida
   RUN_TYPE GEO_OPT
   PRINT_LEVEL LOW
&END GLOBAL
