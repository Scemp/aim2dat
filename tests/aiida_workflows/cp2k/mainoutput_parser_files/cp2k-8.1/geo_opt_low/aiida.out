 DBCSR| CPU Multiplication driver                                           XSMM
 DBCSR| Multrec recursion limit                                              512
 DBCSR| Multiplication stack size                                           1000
 DBCSR| Maximum elements for images                                    UNLIMITED
 DBCSR| Multiplicative factor virtual images                                   1
 DBCSR| Use multiplication densification                                       T
 DBCSR| Multiplication size stacks                                             3
 DBCSR| Use memory pool for CPU allocation                                     F
 DBCSR| Number of 3D layers                                               SINGLE
 DBCSR| Use MPI memory allocation                                              F
 DBCSR| Use RMA algorithm                                                      F
 DBCSR| Use Communication thread                                               T
 DBCSR| Communication thread load                                             83
 DBCSR| MPI: My node id                                                        0
 DBCSR| MPI: Number of nodes                                                  24
 DBCSR| OMP: Current number of threads                                         2
 DBCSR| OMP: Max number of threads                                             2
 DBCSR| Split modifier for TAS multiplication algorithm                  1.0E+00


  **** **** ******  **  PROGRAM STARTED AT               2021-05-24 14:08:46.435
 ***** ** ***  *** **   PROGRAM STARTED ON                               mpcl111
 **    ****   ******    PROGRAM STARTED BY                              pubo1141
 ***** **    ** ** **   PROGRAM PROCESS ID                                 17480
  **** **  *******  **  PROGRAM STARTED IN /user/pubo1141/simulations/aiida_test
                                           s/cp2k_8.1/geo_opt_low

 CP2K| version string:                                          CP2K version 8.1
 CP2K| source code revision number:                                  git:0b61f2f
 CP2K| cp2kflags: omp libint fftw3 libxc parallel mpi3 scalapack xsmm spglib mkl
 CP2K|             libvori libbqb
 CP2K| is freely available from                            https://www.cp2k.org/
 CP2K| Program compiled at                         Sun May 23 19:42:40 CEST 2021
 CP2K| Program compiled on                                               hpcl002
 CP2K| Program compiled for                                        carl-mkl-ompi
 CP2K| Data directory path            /user/pubo1141/software/cp2k/cp2k-8.1/data
 CP2K| Input file name                                                 aiida.inp

 GLOBAL| Force Environment number                                              1
 GLOBAL| Basis set file name                                        BASIS_MOLOPT
 GLOBAL| Potential file name                                      GTH_POTENTIALS
 GLOBAL| MM Potential file name                                     MM_POTENTIAL
 GLOBAL| Coordinate file name                                   aiida.coords.xyz
 GLOBAL| Method name                                                        CP2K
 GLOBAL| Project name                                                      aiida
 GLOBAL| Preferred FFT library                                             FFTW3
 GLOBAL| Preferred diagonalization lib.                                       SL
 GLOBAL| Run type                                                        GEO_OPT
 GLOBAL| All-to-all communication in single precision                          F
 GLOBAL| FFTs using library dependent lengths                                  F
 GLOBAL| Global print level                                                  LOW
 GLOBAL| MPI I/O enabled                                                       T
 GLOBAL| Total number of message passing processes                            24
 GLOBAL| Number of threads for this process                                    2
 GLOBAL| This output is from process                                           0
 GLOBAL| CPU model name                Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz
 GLOBAL| CPUID                                                              1002
 GLOBAL| Compiled for CPUID                                                    0

 *** HINT in environment.F:802 :: The compiler target flags (generic) used ***
 *** to build this binary cannot exploit all extensions of this CPU model  ***
 *** (x86_avx2). Consider compiler target flags as part of FCFLAGS and     ***
 *** CFLAGS (ARCH file).                                                   ***


 MEMORY| system memory details [Kb]
 MEMORY|                        rank 0           min           max       average
 MEMORY| MemTotal            131594208     131594208     131594208     131594208
 MEMORY| MemFree              92310216      62311704      92310216      77311903
 MEMORY| Buffers                167652        159092        167652        163372
 MEMORY| Cached               28286788      28286788      55856816      42071802
 MEMORY| Slab                  1661748       1661748       3839752       2750750
 MEMORY| SReclaimable           898732        898732       2842524       1870628
 MEMORY| MemLikelyFree       121663388     121170136     121663388     121417705


 GENERATE|  Preliminary Number of Bonds generated:                             0
 GENERATE|  Achieved consistency in connectivity generation.

 *******************************************************************************
 *******************************************************************************
 **                                                                           **
 **     #####                         ##              ##                      **
 **    ##   ##            ##          ##              ##                      **
 **   ##     ##                       ##            ######                    **
 **   ##     ##  ##   ##  ##   #####  ##  ##   ####   ##    #####    #####    **
 **   ##     ##  ##   ##  ##  ##      ## ##   ##      ##   ##   ##  ##   ##   **
 **   ##  ## ##  ##   ##  ##  ##      ####     ###    ##   ######   ######    **
 **    ##  ###   ##   ##  ##  ##      ## ##      ##   ##   ##       ##        **
 **     #######   #####   ##   #####  ##  ##  ####    ##    #####   ##        **
 **           ##                                                    ##        **
 **                                                                           **
 **                                                ... make the atoms dance   **
 **                                                                           **
 **            Copyright (C) by CP2K developers group (2000 - 2020)           **
 **                      J. Chem. Phys. 152, 194103 (2020)                    **
 **                                                                           **
 *******************************************************************************


 TOTAL NUMBERS AND MAXIMUM NUMBERS

  Total number of            - Atomic kinds:                                   1
                             - Atoms:                                          2
                             - Shell sets:                                     2
                             - Shells:                                         6
                             - Primitive Cartesian functions:                 10
                             - Cartesian basis functions:                     10
                             - Spherical basis functions:                     10

  Maximum angular momentum of- Orbital basis functions:                        1
                             - Local part of the GTH pseudopotential:          2


 SCF PARAMETERS         Density guess:                                    ATOMIC
                        --------------------------------------------------------
                        max_scf:                                              50
                        max_scf_history:                                       0
                        max_diis:                                              4
                        --------------------------------------------------------
                        eps_scf:                                        1.00E-05
                        eps_scf_history:                                0.00E+00
                        eps_diis:                                       1.00E-01
                        eps_eigval:                                     1.00E-05
                        --------------------------------------------------------
                        level_shift [a.u.]:                                 0.00
                        --------------------------------------------------------
                        Mixing method:                           DIRECT_P_MIXING
                        --------------------------------------------------------
                        No outer SCF

 BFGS| Use rational function optimization for step estimation:                NO
 BFGS| Use model Hessian for initial guess:                                  YES
 BFGS| Restart Hessian:                                                       NO
 BFGS| Trust radius:                                                       0.472

 *******************************************************************************
 ***                     STARTING GEOMETRY OPTIMIZATION                      ***
 ***                                   BFGS                                  ***
 *******************************************************************************

 Number of electrons:                                                          2
 Number of occupied orbitals:                                                  1
 Number of molecular orbitals:                                                 1

 Number of orbital functions:                                                 10
 Number of independent orbital functions:                                     10

 Extrapolation method: initial_guess


 SCF WAVEFUNCTION OPTIMIZATION

  Step     Update method      Time    Convergence         Total energy    Change
  ------------------------------------------------------------------------------
     1 P_Mix/Diag. 0.40E+00    0.0     0.58131095        -0.8981598320 -8.98E-01
     2 P_Mix/Diag. 0.40E+00    0.0     0.34923638        -1.0032835397 -1.05E-01
     3 P_Mix/Diag. 0.40E+00    0.0     0.20976519        -1.0663496444 -6.31E-02
     4 P_Mix/Diag. 0.40E+00    0.0     0.12596517        -1.1041905476 -3.78E-02
     5 P_Mix/Diag. 0.40E+00    0.0     0.07562951        -1.1268968137 -2.27E-02
     6 DIIS/Diag.  0.53E-03    0.0     0.04542519        -1.1405215434 -1.36E-02
     7 DIIS/Diag.  0.48E-04    0.0     0.00001007        -1.1609605120 -2.04E-02
     8 DIIS/Diag.  0.76E-04    0.0     0.00000860        -1.1609605117  2.94E-10

  *** SCF run converged in     8 steps ***


  Electronic density on regular grids:         -1.9999999994        0.0000000006
  Core density on regular grids:                1.9999999999       -0.0000000001
  Total charge density on r-space grids:        0.0000000005
  Total charge density g-space grids:           0.0000000005

  Overlap energy of the core charge distribution:               0.00000060512959
  Self energy of the core charge distribution:                 -2.82094791773878
  Core Hamiltonian energy:                                      1.07883183773296
  Hartree energy:                                               1.30392538016458
  Exchange-correlation energy:                                 -0.72277041696069

  Total energy:                                                -1.16096051167235

 ENERGY| Total FORCE_EVAL ( QS ) energy [a.u.]:               -1.160960512083268

 --------  Informations at step =     0 ------------
  Optimization Method        =                 BFGS
  Total Energy               =        -1.1609605121
  Used time                  =                0.332
 ---------------------------------------------------

 --------------------------
 OPTIMIZATION STEP:      1
 --------------------------

 Number of electrons:                                                          2
 Number of occupied orbitals:                                                  1
 Number of molecular orbitals:                                                 1

 Number of orbital functions:                                                 10
 Number of independent orbital functions:                                     10

 Extrapolation method: PS Nth order
 Extrapolation order:  0


 SCF WAVEFUNCTION OPTIMIZATION

  Step     Update method      Time    Convergence         Total energy    Change
  ------------------------------------------------------------------------------
     1 P_Mix/Diag. 0.40E+00    0.0     0.00061766        -1.1611751777 -1.16E+00
     2 P_Mix/Diag. 0.40E+00    0.0     0.00038032        -1.1611771318 -1.95E-06
     3 DIIS/Diag.  0.68E-04    0.0     0.00024387        -1.1611783352 -1.20E-06
     4 DIIS/Diag.  0.45E-06    0.0     0.00000025        -1.1611801877 -1.85E-06

  *** SCF run converged in     4 steps ***


  Electronic density on regular grids:         -1.9999999984        0.0000000016
  Core density on regular grids:                1.9999999996       -0.0000000004
  Total charge density on r-space grids:        0.0000000012
  Total charge density g-space grids:           0.0000000012

  Overlap energy of the core charge distribution:               0.00000117383417
  Self energy of the core charge distribution:                 -2.82094791773878
  Core Hamiltonian energy:                                      1.08370606372530
  Hartree energy:                                               1.30183473613795
  Exchange-correlation energy:                                 -0.72577424368122

  Total energy:                                                -1.16118018772259

 ENERGY| Total FORCE_EVAL ( QS ) energy [a.u.]:               -1.161180187722721

 --------  Informations at step =     1 ------------
  Optimization Method        =                 BFGS
  Total Energy               =        -1.1611801877
  Real energy change         =        -0.0002196756
  Predicted change in energy =        -0.0002539133
  Scaling factor             =         0.0000000000
  Step size                  =         0.0178180049
  Trust radius               =         0.4724315332
  Decrease in energy         =                  YES
  Used time                  =                0.107

  Convergence check :
  Max. step size             =         0.0178180049
  Conv. limit for step size  =         0.0030000000
  Convergence in step size   =                   NO
  RMS step size              =         0.0102872299
  Conv. limit for RMS step   =         0.0015000000
  Convergence in RMS step    =                   NO
  Max. gradient              =         0.0022539017
  Conv. limit for gradients  =         0.0004500000
  Conv. for gradients        =                   NO
  RMS gradient               =         0.0013012907
  Conv. limit for RMS grad.  =         0.0003000000
  Conv. for gradients        =                   NO
 ---------------------------------------------------

 --------------------------
 OPTIMIZATION STEP:      2
 --------------------------

 Number of electrons:                                                          2
 Number of occupied orbitals:                                                  1
 Number of molecular orbitals:                                                 1

 Number of orbital functions:                                                 10
 Number of independent orbital functions:                                     10

 Extrapolation method: PS Nth order
 Extrapolation order:  1


 SCF WAVEFUNCTION OPTIMIZATION

  Step     Update method      Time    Convergence         Total energy    Change
  ------------------------------------------------------------------------------
     1 P_Mix/Diag. 0.40E+00    0.0     0.00070394        -1.1611788527 -1.16E+00
     2 P_Mix/Diag. 0.40E+00    0.0     0.00043348        -1.1611813944 -2.54E-06
     3 DIIS/Diag.  0.78E-04    0.0     0.00027802        -1.1611829598 -1.57E-06
     4 DIIS/Diag.  0.54E-06    0.0     0.00000032        -1.1611853698 -2.41E-06

  *** SCF run converged in     4 steps ***


  Electronic density on regular grids:         -1.9999999985        0.0000000015
  Core density on regular grids:                1.9999999997       -0.0000000003
  Total charge density on r-space grids:        0.0000000011
  Total charge density g-space grids:           0.0000000011

  Overlap energy of the core charge distribution:               0.00000107320893
  Self energy of the core charge distribution:                 -2.82094791773878
  Core Hamiltonian energy:                                      1.08302261158443
  Hartree energy:                                               1.30210002919288
  Exchange-correlation energy:                                 -0.72536116601326

  Total energy:                                                -1.16118536976580

 ENERGY| Total FORCE_EVAL ( QS ) energy [a.u.]:               -1.161185369765988

 --------  Informations at step =     2 ------------
  Optimization Method        =                 BFGS
  Total Energy               =        -1.1611853698
  Real energy change         =        -0.0000051820
  Predicted change in energy =        -0.0000054844
  Scaling factor             =         0.0000000000
  Step size                  =         0.0024333103
  Trust radius               =         0.4724315332
  Decrease in energy         =                  YES
  Used time                  =                0.126

  Convergence check :
  Max. step size             =         0.0024333103
  Conv. limit for step size  =         0.0030000000
  Convergence in step size   =                  YES
  RMS step size              =         0.0014048724
  Conv. limit for RMS step   =         0.0015000000
  Convergence in RMS step    =                  YES
  Max. gradient              =         0.0001178855
  Conv. limit for gradients  =         0.0004500000
  Conv. in gradients         =                  YES
  RMS gradient               =         0.0000680612
  Conv. limit for RMS grad.  =         0.0003000000
  Conv. in RMS gradients     =                  YES
 ---------------------------------------------------

 *******************************************************************************
 ***                    GEOMETRY OPTIMIZATION COMPLETED                      ***
 *******************************************************************************

                    Reevaluating energy at the minimum

 Number of electrons:                                                          2
 Number of occupied orbitals:                                                  1
 Number of molecular orbitals:                                                 1

 Number of orbital functions:                                                 10
 Number of independent orbital functions:                                     10

 Extrapolation method: PS Nth order
 Extrapolation order:  2


 SCF WAVEFUNCTION OPTIMIZATION

  Step     Update method      Time    Convergence         Total energy    Change
  ------------------------------------------------------------------------------
     1 P_Mix/Diag. 0.40E+00    0.0     0.00079045        -1.1611771560 -1.16E+00
     2 P_Mix/Diag. 0.40E+00    0.0     0.00048677        -1.1611803593 -3.20E-06
     3 DIIS/Diag.  0.88E-04    0.0     0.00031222        -1.1611823322 -1.97E-06
     4 DIIS/Diag.  0.53E-06    0.0     0.00000030        -1.1611853698 -3.04E-06

  *** SCF run converged in     4 steps ***


  Electronic density on regular grids:         -1.9999999985        0.0000000015
  Core density on regular grids:                1.9999999997       -0.0000000003
  Total charge density on r-space grids:        0.0000000011
  Total charge density g-space grids:           0.0000000011

  Overlap energy of the core charge distribution:               0.00000107320893
  Self energy of the core charge distribution:                 -2.82094791773878
  Core Hamiltonian energy:                                      1.08302279969940
  Hartree energy:                                               1.30209993733485
  Exchange-correlation energy:                                 -0.72536126227020

  Total energy:                                                -1.16118536976581

 ENERGY| Total FORCE_EVAL ( QS ) energy [a.u.]:               -1.161185369765809

 -------------------------------------------------------------------------------
 -                                                                             -
 -                                DBCSR STATISTICS                             -
 -                                                                             -
 -------------------------------------------------------------------------------
 COUNTER                                    TOTAL       BLAS       SMM       ACC
 flops     5 x     1 x     5                  600       0.0%    100.0%      0.0%
 flops inhomo. stacks                        3900     100.0%      0.0%      0.0%
 flops total                         4.500000E+03      86.7%     13.3%      0.0%
 flops max/rank                      4.500000E+03      86.7%     13.3%      0.0%
 matmuls inhomo. stacks                        78     100.0%      0.0%      0.0%
 matmuls total                                 90      86.7%     13.3%      0.0%
 number of processed stacks                    32      81.2%     18.8%      0.0%
 average stack size                                     3.0       2.0       0.0
 marketing flops                     5.799999E+03
 -------------------------------------------------------------------------------
 # multiplications                             29
 max memory usage/rank             239.820800E+06
 # max total images/rank                        3
 # max 3D layers                                1
 # MPI messages exchanged                   13224
 MPI messages size (bytes):
  total size                        29.360000E+03
  min size                           0.000000E+00
  max size                         400.000000E+00
  average size                       2.220206E+00
 MPI breakdown and total messages size (bytes):
             size <=      128               13194                    17360
       128 < size <=     8192                  30                    12000
      8192 < size <=    32768                   0                        0
     32768 < size <=   131072                   0                        0
    131072 < size <=  4194304                   0                        0
   4194304 < size <= 16777216                   0                        0
  16777216 < size                               0                        0
 -------------------------------------------------------------------------------

 *** WARNING in dbcsr_mm.F:294 :: Using a non-square number of MPI ranks ***
 *** might lead to poor performance. Used ranks: 24 Suggested: 25 49     ***

 -------------------------------------------------------------------------------
 -                                                                             -
 -                      DBCSR MESSAGE PASSING PERFORMANCE                      -
 -                                                                             -
 -------------------------------------------------------------------------------
 ROUTINE             CALLS      AVE VOLUME [Bytes]
 MP_Bcast                2                     12.
 MP_Allreduce          207                      9.
 MP_Alltoall           798                   1724.
 MP_ISend             1218                     68.
 MP_IRecv             1218                     61.
 -------------------------------------------------------------------------------

 -------------------------------------------------------------------------------
 -                                                                             -
 -                                GRID STATISTICS                              -
 -                                                                             -
 -------------------------------------------------------------------------------
 LP    KERNEL             BACKEND                              COUNT     PERCENT
 2     collocate ortho    REF                                   1800      50.86%
 2     integrate ortho    REF                                   1500      42.38%
 3     integrate ortho    REF                                    225       6.36%
 0     collocate general  REF                                      8       0.23%
 1     integrate general  REF                                      6       0.17%
 -------------------------------------------------------------------------------

 MEMORY| Estimated peak process memory [MiB]                                 229

 -------------------------------------------------------------------------------
 -                                                                             -
 -                         MESSAGE PASSING PERFORMANCE                         -
 -                                                                             -
 -------------------------------------------------------------------------------

 ROUTINE             CALLS      AVE VOLUME [Bytes]
 MP_Group                5
 MP_Bcast               80                2301186.
 MP_Allreduce          738                    136.
 MP_Sync               121
 MP_Alltoall           450                2203949.
 MP_ISendRecv         4485                  11236.
 MP_Wait              4579
 MP_comm_split          23
 MP_ISend              496                     43.
 MP_IRecv              491                     43.
 MP_Recv                10                     48.
 -------------------------------------------------------------------------------


 -------------------------------------------------------------------------------
 -                                                                             -
 -                           R E F E R E N C E S                               -
 -                                                                             -
 -------------------------------------------------------------------------------
 
 CP2K version 8.1, the CP2K developers group (2020).
 CP2K is freely available from https://www.cp2k.org/ .

 Kuehne,Thomas D.; Iannuzzi,Marcella; Del Ben,Mauro;
 Rybkin,Vladimir V.; Seewald,Patrick; Stein,Frederick; Laino,Teodoro;
 Khaliullin,Rustam Z.; Schuett,Ole; Schiffmann,Florian; Golze,Dorothea;
 Wilhelm,Jan; Chulkov,Sergey; Bani-Hashemian,Mohammad Hossein;
 Weber,Valery; Borstnik,Urban; Taillefumier,Mathieu;
 Jakobovits,Alice Shoshana; Lazzaro,Alfio; Pabst,Hans; Mueller,Tiziano;
 Schade,Robert; Guidon,Manuel; Andermatt,Samuel; Holmberg,Nico;
 Schenter,Gregory K.; Hehn,Anna; Bussy,Augustin; Belleflamme,Fabian;
 Tabacchi,Gloria; Gloess,Andreas; Lass,Michael; Bethune,Iain;
 Mundy,Christopher J.; Plessl,Christian; Watkins,Matt;
 VandeVondele,Joost; Krack,Matthias; Hutter,Juerg. 
 The Journal of Chemical Physics, 152 (19), (2020). 
 CP2K: An electronic structure and molecular dynamics software package -
 Quickstep: Efficient and accurate electronic structure calculations.
 https://doi.org/10.1063/5.0007045

 Schuett, Ole; Messmer, Peter; Hutter, Juerg; VandeVondele, Joost. 
 Electronic Structure Calculations on Graphics Processing Units, John
 Wiley & Sons, Ltd, 173-190 (2016). 
 GPU-Accelerated Sparse Matrix-Matrix Multiplication for
 Linear Scaling Density Functional Theory.
 https://doi.org/10.1002/9781118670712.ch8

 Borstnik, U; VandeVondele, J; Weber, V; Hutter, J. 
 PARALLEL COMPUTING, 40 (5-6), 47-58 (2014). 
 Sparse matrix multiplication: The distributed block-compressed sparse
 row library.
 https://doi.org/10.1016/j.parco.2014.03.012

 Hutter, J; Iannuzzi, M; Schiffmann, F; VandeVondele, J. 
 WIREs Comput Mol Sci., 4 (1), 15-25 (2014). 
 CP2K: atomistic simulations of condensed matter systems.
 https://doi.org/10.1002/wcms.1159

 VandeVondele, J; Hutter, J. 
 JOURNAL OF CHEMICAL PHYSICS, 127 (11), 114105 (2007). 
 Gaussian basis sets for accurate calculations on molecular systems in
 gas and condensed phases.
 https://doi.org/10.1063/1.2770708

 Krack, M. 
 THEORETICAL CHEMISTRY ACCOUNTS, 114 (1-3), 145-152 (2005). 
 Pseudopotentials for H to Kr optimized for gradient-corrected
 exchange-correlation functionals.
 https://doi.org/10.1007/s00214-005-0655-y

 VandeVondele, J; Krack, M; Mohamed, F; Parrinello, M; Chassaing, T;
 Hutter, J. COMPUTER PHYSICS COMMUNICATIONS, 167 (2), 103-128 (2005). 
 QUICKSTEP: Fast and accurate density functional calculations using a
 mixed Gaussian and plane waves approach.
 https://doi.org/10.1016/j.cpc.2004.12.014

 Frigo, M; Johnson, SG. 
 PROCEEDINGS OF THE IEEE, 93 (2), 216-231 (2005). 
 The design and implementation of FFTW3.
 https://doi.org/10.1109/JPROC.2004.840301

 Martyna, GJ; Tuckerman, ME. 
 JOURNAL OF CHEMICAL PHYSICS, 110 (6), 2810-2821 (1999). 
 A reciprocal space based method for treating long range interactions in
 ab initio and force-field-based calculations in clusters.
 https://doi.org/10.1063/1.477923

 Hartwigsen, C; Goedecker, S; Hutter, J. 
 PHYSICAL REVIEW B, 58 (7), 3641-3662 (1998). 
 Relativistic separable dual-space Gaussian pseudopotentials from H to Rn.
 https://doi.org/10.1103/PhysRevB.58.3641

 Lippert, G; Hutter, J; Parrinello, M. 
 MOLECULAR PHYSICS, 92 (3), 477-487 (1997). 
 A hybrid Gaussian and plane wave density functional scheme.
 https://doi.org/10.1080/002689797170220

 Perdew, JP; Burke, K; Ernzerhof, M. 
 PHYSICAL REVIEW LETTERS, 77 (18), 3865-3868 (1996). 
 Generalized gradient approximation made simple.
 https://doi.org/10.1103/PhysRevLett.77.3865

 Goedecker, S; Teter, M; Hutter, J. 
 PHYSICAL REVIEW B, 54 (3), 1703-1710 (1996). 
 Separable dual-space Gaussian pseudopotentials.
 https://doi.org/10.1103/PhysRevB.54.1703


 -------------------------------------------------------------------------------
 -                                                                             -
 -                                T I M I N G                                  -
 -                                                                             -
 -------------------------------------------------------------------------------
 SUBROUTINE                       CALLS  ASD         SELF TIME        TOTAL TIME
                                MAXIMUM       AVERAGE  MAXIMUM  AVERAGE  MAXIMUM
 CP2K                                 1  1.0    0.005    0.024    0.828    0.832
 cp_geo_opt                           1  2.0    0.000    0.000    0.694    0.697
 geoopt_bfgs                          1  3.0    0.001    0.001    0.694    0.697
 cp_eval_at                           4  4.0    0.000    0.001    0.608    0.614
 qs_energies                          4  5.8    0.000    0.000    0.573    0.579
 qs_forces                            3  5.0    0.000    0.000    0.491    0.496
 scf_env_do_scf                       4  6.8    0.000    0.001    0.422    0.428
 scf_env_do_scf_inner_loop           20  7.8    0.001    0.004    0.422    0.427
 rebuild_ks_matrix                   23  9.4    0.000    0.000    0.192    0.221
 qs_ks_build_kohn_sham_matrix        23 10.4    0.002    0.002    0.192    0.221
 pw_transfer                        447 13.2    0.009    0.009    0.199    0.208
 fft_wrap_pw1pw2                    401 14.2    0.001    0.002    0.189    0.198
 qs_ks_update_qs_env                 20  8.8    0.000    0.000    0.168    0.191
 fft3d_ps                           401 16.2    0.035    0.039    0.167    0.178
 fft_wrap_pw1pw2_140                259 15.3    0.006    0.008    0.162    0.171
 mp_alltoall_z22v                   401 18.2    0.103    0.138    0.103    0.138
 rs_pw_transfer                     195 13.0    0.001    0.001    0.123    0.135
 qs_rho_update_rho                   24  8.9    0.000    0.000    0.125    0.125
 calculate_rho_elec                  24  9.9    0.001    0.001    0.125    0.125
 qs_scf_new_mos                      20  8.8    0.000    0.000    0.116    0.118
 qs_vxc_create                       23 11.4    0.000    0.000    0.109    0.111
 xc_vxc_pw_create                    23 12.4    0.002    0.002    0.109    0.111
 density_rs2pw                       24 10.9    0.001    0.001    0.100    0.110
 mp_waitall_1                      6203 15.3    0.083    0.098    0.083    0.098
 yz_to_x                            145 17.0    0.005    0.006    0.084    0.097
 qs_energies_init_hamiltonians        4  6.8    0.000    0.000    0.087    0.093
 qs_env_update_s_mstruct              4  7.8    0.000    0.000    0.082    0.089
 calculate_rho_core                   4  8.8    0.000    0.000    0.082    0.089
 rs_pw_transfer_RS2PW_140            28 12.6    0.010    0.011    0.071    0.087
 sum_up_and_integrate                23 11.4    0.001    0.001    0.058    0.087
 integrate_v_rspace                  23 12.4    0.000    0.002    0.058    0.086
 cp_fm_syevd                         23  9.9    0.000    0.000    0.074    0.075
 xc_rho_set_and_dset_create          23 13.4    0.001    0.002    0.063    0.067
 get_fft_scratch                    402 17.2    0.033    0.063    0.034    0.064
 fft3d_s                              1  2.0    0.000    0.000    0.032    0.062
 parser_read_line                  2628  4.0    0.001    0.001    0.040    0.062
 parser_read_line_low                 5  5.4    0.003    0.007    0.039    0.061
 copy_dbcsr_to_fm                    43 10.0    0.001    0.001    0.057    0.059
 broadcast_input_information          5  6.4    0.000    0.000    0.036    0.059
 cp_fm_redistribute_start            23 10.9    0.044    0.051    0.049    0.057
 potential_pw2rs                     23 13.4    0.001    0.001    0.055    0.056
 init_scf_run                         4  6.8    0.000    0.001    0.049    0.050
 grid_collocate_task_list            24 10.9    0.023    0.047    0.023    0.047
 cp_dbcsr_plus_fm_fm_t_native        26 10.6    0.000    0.000    0.045    0.047
 cp_fm_write_unformatted              6  8.8    0.045    0.047    0.045    0.047
 create_qs_kind_set                   1  2.0    0.000    0.000    0.031    0.047
 read_qs_kind                         1  3.0    0.011    0.015    0.031    0.047
 mp_alltoall_i22                    118 12.4    0.040    0.045    0.040    0.045
 mp_bcast_iv                         12  6.9    0.014    0.045    0.014    0.045
 topology_control                     1  2.0    0.000    0.000    0.021    0.045
 coordinate_control                   1  3.0    0.000    0.001    0.020    0.044
 x_to_yz                            162 17.5    0.012    0.015    0.035    0.044
 coordinate_control_READ_COORDI       1  4.0    0.000    0.000    0.020    0.043
 read_coordinate_xyz                  1  5.0    0.000    0.001    0.020    0.043
 mp_bcast_i                          38  4.8    0.012    0.040    0.012    0.040
 dbcsr_desymmetrize_deep             43 11.0    0.001    0.001    0.036    0.039
 scf_env_initial_rho_setup            4  7.8    0.000    0.001    0.038    0.039
 write_mo_set_to_restart             20  8.8    0.002    0.037    0.032    0.038
 calculate_dm_sparse                 23  9.8    0.000    0.000    0.035    0.037
 dbcsr_multiply_generic              29 11.6    0.001    0.001    0.035    0.036
 eigensolver                         20  9.8    0.001    0.001    0.032    0.033
 xc_functional_eval                  23 14.4    0.000    0.000    0.028    0.031
 pbe_lda_eval                        23 15.4    0.028    0.031    0.028    0.031
 dbcsr_complete_redistribute         75 11.6    0.002    0.003    0.030    0.031
 write_mo_set_low                     4  9.8    0.000    0.000    0.030    0.031
 grid_integrate_task_list            23 13.4    0.002    0.031    0.002    0.031
 qs_ks_update_qs_env_forces           3  6.0    0.000    0.000    0.026    0.031
 mp_bcast_am                          5  7.4    0.012    0.029    0.014    0.029
 cp_fm_redistribute_end              23 10.9    0.022    0.026    0.022    0.027
 rs_pw_transfer_PW2RS_140            26 14.9    0.014    0.016    0.023    0.027
 wfi_extrapolate                      4  8.8    0.000    0.000    0.026    0.026
 qs_init_subsys                       1  2.0    0.001    0.002    0.026    0.026
 qs_env_setup                         1  3.0    0.000    0.000    0.024    0.025
 qs_env_rebuild_pw_env                8  6.1    0.000    0.000    0.024    0.025
 pw_env_rebuild                       1  5.0    0.000    0.001    0.024    0.025
 write_restart                        3  4.0    0.001    0.024    0.001    0.024
 mp_sum_d                           608  4.5    0.014    0.021    0.014    0.021
 mp_sum_l                           490  5.1    0.015    0.018    0.015    0.018
 -------------------------------------------------------------------------------

 The number of warnings for this run is : 1
 
 -------------------------------------------------------------------------------
  **** **** ******  **  PROGRAM ENDED AT                 2021-05-24 14:08:48.581
 ***** ** ***  *** **   PROGRAM RAN ON                                   mpcl111
 **    ****   ******    PROGRAM RAN BY                                  pubo1141
 ***** **    ** ** **   PROGRAM PROCESS ID                                 17480
  **** **  *******  **  PROGRAM STOPPED IN /user/pubo1141/simulations/aiida_test
                                           s/cp2k_8.1/geo_opt_low
